<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1221</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1221</width>
       <height>743</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone1 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="cityStateGroup">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>0</y>
        <width>1201</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color:rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>Select Location:</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="stateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>55</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="cityLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>34</y>
         <width>60</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>59</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeLabel">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>33</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipcodeList">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>101</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="categoryLabel">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>33</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter By Category:</string>
       </property>
      </widget>
      <widget class="QListWidget" name="categoryList">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>60</y>
         <width>221</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeLabel_4">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode Statistics:</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeLabel_5">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string># of Businesses:</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeLabel_6">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>100</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Total Population:</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeLabel_7">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>140</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Average Income:</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeLabel_8">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>180</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Median Income:</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeNumBusinessesLabel">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>60</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodePopulationLabel">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>100</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeAvgIncomeLabel">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>140</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeMedianIncomeLabel">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>180</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>21</width>
         <height>221</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>950</x>
         <y>10</y>
         <width>21</width>
         <height>221</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="categoryLabel_2">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>30</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Top Categories in Zipcode:</string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>10</y>
         <width>21</width>
         <height>221</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QTableWidget" name="topCategories">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>60</y>
         <width>291</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>240</y>
        <width>1200</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color:rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>1161</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTableWidget" name="popularBusinessTable">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>241</y>
         <width>581</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTableWidget" name="successfulBusinessTable">
       <property name="geometry">
        <rect>
         <x>605</x>
         <y>240</y>
         <width>581</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QLabel" name="zipcodeLabel_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Popular Businesses (in zipcode)</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipcodeLabel_3">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>210</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Successful Businesses (in zipcode)</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="databasePW">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>1</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>6</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Database Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterPW">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
